README.md

```diff
@@ -197,5 +194,14 @@ make docker/Dockerfile.b6
 docker build --build-arg=TARGETOS=linux --build-arg=TARGETARCH=amd64 -f docker/Dockerfile.b6 .
 ```
 
+## World Structure
+
+
+![The World Structure](python/docs/b6-world.png)
+
+Data in the world is organised in identifiable features, which can represent anything from a point to an analysis result, with information encoded via tag key/values pairs - e.g. a path mapping to a list of lat/lngs representing the points that belong to it. The initial loaded world is in a compact format encoding, to enable fast search, but, any change done to the world, be it adding a new feature, or editting an existing one, by e.g. adding a tag to it, or changing its ID, is applied in an overlay layer, that is not compact. 
+
 
+![The Compact World](python/docs/b6-compact-world.png)
 
+ The compact form of the world enables fast search by minimising memory consumption of data that needs to be available for fetching results. This is done via custom binary encodings, that account for underlying data types. In addition to the feature lookup table, there are mapping tables that optimise general search - one handling string encodings (e.g. "highway"), and one optimising token (e.g. "highway=primary") lookup.
```

Comment:
I feel we need to say why the compact encoding is fast, eg instead of:
_enables fast search by minimising memory consumption of data_
maybe:
_enables fast search by minimising memory consumption, allowing large spatial areas to be represented entirely in memory on a single machine_.